src/components/ui/Radio.tsx

```diff
@@ -47,7 +48,8 @@ class Radio extends Component<Props> {
           </label>
         )}
         <div className="franz-form__radio-wrapper">
-          {field.options.map(type => (
+          {/* @ts-expect-error Property 'map' does not exist on type 'OptionsModel'. */}
+          {field.options?.map(type => (
```

Comment:
I think an arbitrary naming of `options` was chosen to be passed in as possible radio options.

Unfortunately, it clashes with the naming of some `mobx-react-form` specific type, but we can ignore it.

Reply:
ok - as long as its working for now, its fine.
But, do you think we should put a `NOTE` or some other comment with this text for future reference?

Reply:
This LOC will always lead back to this PR, so that's also one way to read up on why the TS error is ignored.